packages/desktop/views/dashboard/send-flow/send-flow-route.enum.ts

```diff
@@ -1,6 +1,6 @@
 export enum SendFlowRoute {
+    SelectAmount = 'selectAmount',
```

Comment:
select amount doesn't make sense, the verb select is not what we are doing. InputAmount makes more sense